DesktopModules/Vanjaro/UXManager/Library/Base.ascx.cs

```diff
@@ -94,10 +94,15 @@ protected void Page_Load(object sender, EventArgs e)
                     }
 
                     WebForms.LinkCSS(Page, "UXManagerAppCss", Page.ResolveUrl("~/DesktopModules/Vanjaro/UXManager/Library/Resources/StyleSheets/app.css"));
-                    WebForms.LinkCSS(Page, "GrapesJsCss", Page.ResolveUrl("~/DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/GrapesJs/css/grapes.min.css"));
-                    WebForms.LinkCSS(Page, "GrapickJsCss", Page.ResolveUrl("~/DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/GrapesJs/css/grapick.min.css"));
+
+                    if (GrapesjsInit())
```

Comment:
GrapesjsInit is not appropriate method name. It should be generic such as ShowUXMananger